packages/integration-karma/test/shadow-dom/Element-properties/Element.id.spec.js

```diff
@@ -9,3 +11,55 @@ it('should allow setting id properties manually', () => {
     shadowRoot.querySelector('div').id = 'something';
     expect(shadowRoot.querySelector('div').getAttribute('id')).toBe('something');
 });
+
+describe('scoped-ids', () => {
+    describe('expressions', () => {
+        describe('custom elements', () => {
+            it('should render expected id attribute value when its value is set to `undefined`', () => {
+                const elm = createElement('x-foo', { is: IdValueUndefined });
+                expect(() => {
+                    document.body.appendChild(elm);
+                }).toLogErrorDev(
+                    /\[LWC error\]: Invalid id value "undefined". The id attribute must contain a non-empty string./
+                );
+                const child = elm.shadowRoot.querySelector('x-child');
+                expect(child.getAttribute('id')).toEqual(null);
+            });
+
+            it('should render the id attribute as a boolean attribute when its value is set to an empty string', () => {
+                const elm = createElement('x-foo', { is: IdValueEmpty });
+                expect(() => {
+                    document.body.appendChild(elm);
+                }).toLogErrorDev(
+                    /\[LWC error\]: Invalid id value "". The id attribute must contain a non-empty string./
+                );
+                const child = elm.shadowRoot.querySelector('x-child');
+                expect(child.getAttribute('id')).toEqual(null);
```

Comment:
Looks like this skipped test was incomplete. `x-child` doesn't have an `id` attribute so we need to add it. I tested it and saw that the value should be an empty string.